app/controllers/api/auth/stories_controller.rb

```diff
@@ -38,16 +38,16 @@ def resources_base
     @stories ||= if params[:network_id]
       super.published
     else
-      current_user.approved_account_stories
+      authorization.token_auth_stories
     end
   end
 
   def create_resource
     super.tap do |story|
       story.creator_id = current_user.id
       story.account_id ||= story.series.try(:account_id)
-      story.account_id ||= current_user.account_id
-      story.account_id ||= current_user.approved_accounts.first.try(:id)
+      story.account_id ||= current_user.account_id if authorization.authorized?(current_user.default_account)
```

Comment:
thanks for adding that check - clever to use the `authorized?` method, I was thinking of doing a detect on the account list - this is much better